src/main/webapp/script.js

```diff
@@ -366,5 +370,114 @@ function addInputDestinationLocationAutofill() {
   locationAutocomplete.addListener('place_changed', () => {
     inputDestination.classList.add('is-valid');
     checkNextButton();
+
+    // Remove any current elements, then get and add the suggested locations.
+    removeSuggestedLocations();
+    const radius = 50000;
+    let location = new google.maps.LatLng(locationAutocomplete.getPlace().geometry.location.lat(), 
+      locationAutocomplete.getPlace().geometry.location.lng());
+    getAndAddSuggestedLocations(location, radius);
+
+  });
+}
+
+// Get suggested locations based on a central location (latitude, longitude) and
+// radius (meters). All suggested locations are of type "tourist attraction".
+// Then, call addSuggestedLocations function.
+function getAndAddSuggestedLocations(centralLocation, radius) {
+  let googlePlacesObject = new google.maps.places.PlacesService(document.createElement('div'));
+
+  let placesRequest = {
+    location: centralLocation,
+    radius: radius,
+    type: 'tourist_attraction'
```

Comment:
No action required, but it could be cool to show multiple categories of suggested POIs by category (for example also showing restaurants, etc.) in the future!

Reply:
Yeah that is a good idea, I like that too! I'll keep in store :)